snapshot.go

```diff
@@ -0,0 +1,70 @@
+package civogo
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type Snapshot struct {
+	ID          string    `json:"id"`
+	InstanceID  string    `json:"instance_id"`
+	Hostname    string    `json:"hostname"`
+	Template    string    `json:"template_id"`
+	Region      string    `json:"region"`
+	Name        string    `json:"name"`
+	Safe        int       `json:"safe"`
+	SizeGB      int       `json:"size_gb"`
```

Comment:
Can you use SizeGigabytes please to match other civogo structures. Was more readable in other places like that